packaging/install/020-ClusterRole.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: strimzi-access-operator
+  labels:
+    app: strimzi-access-operator
+rules:
+  - apiGroups:
+      - "kafka.strimzi.io"
+    resources:
+      - kafkaaccesses
+      - kafkaaccesses/status
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - delete
+      - patch
+      - update
+  - apiGroups:
+      - "kafka.strimzi.io"
+    resources:
+      - kafkas
+      - kafkas/status
+      - kafkausers
+      - kafkausers/status
```

Comment:
Do you really get / list / watch status? I think you might not need them here, or?

Reply:
We need to be able to view the status of both the Kafka and KafkaUser to pick out some of the credentials. For example bootstrap server addresses from Kafka and secret name from KafkaUser. So I think for that we do need get, however probably not list or watch?

Reply:
You IMHO do not need that to get status. Only to for example update status. The status is part of the resource. So when you do get on the KafkaUSer or Kafka resource, you get the metadata, spec and status sections. Only when updating, you have to call separate endpoint to update the status.

Reply:
I've double checked and yes I can remove the status parts of the ClusterRole and the operator is still able to view the status of the resources so I've removed them